gluoncv/data/otb/tracking.py

```diff
@@ -0,0 +1,276 @@
+                elif self.name == 'Skating2-1':
+                    txt_name = 'skating2_1.txt'
+                elif self.name == 'Skating2-2':
+                    txt_name = 'skating2_2.txt'
+                elif self.name == 'FaceOcc1':
+                    txt_name = 'faceocc1.txt'
+                elif self.name == 'FaceOcc2':
+                    txt_name = 'faceocc2.txt'
+                elif self.name == 'Human4-2':
+                    txt_name = 'human4_2.txt'
+                else:
+                    txt_name = self.name[0].lower()+self.name[1:]+'.txt'
+                traj_file = os.path.join(path, name, txt_name)
+            if os.path.exists(traj_file):
+                with open(traj_file, 'r') as f :
+                    pred_traj = [list(map(float, x.strip().split(',')))
+                            for x in f.readlines()]
+                    if len(pred_traj) != len(self.gt_traj):
+                        print(name, len(pred_traj), len(self.gt_traj), self.name)
+                    if store:
+                        self.pred_trajs[name] = pred_traj
+                    else:
+                        return pred_traj
+            else:
+                print(traj_file)
+        self.tracker_names = list(self.pred_trajs.keys())
+
+
+
+class OTBTracking(Dataset):
+    """
+    Args:
+        name: dataset name, should be 'OTB100', 'CVPR13', 'OTB50'
+        dataset_root: dataset root
+        load_img: wether to load all imgs
+    """
+    def __init__(self, name, dataset_root, load_img=False):
+        super(OTBTracking, self).__init__(name, dataset_root)
+        with open(os.path.join(dataset_root, name+'.json'), 'r') as f:
+            meta_data = json.load(f)
+
+        # load videos
+        pbar = tqdm(meta_data.keys(), desc='loading '+name, ncols=100)
+        self.videos = {}
+        for video in pbar:
+            pbar.set_postfix_str(video)
+            self.videos[video] = OTBVideo(video,
+                                          dataset_root,
+                                          meta_data[video]['video_dir'],
+                                          meta_data[video]['init_rect'],
+                                          meta_data[video]['img_names'],
+                                          meta_data[video]['gt_rect'],
+                                          meta_data[video]['attr'],
+                                          load_img)
+        # set attr
+        attr = []
+        for x in self.videos.values():
+            attr += x.attr
+        attr = set(attr)
+        self.attr = {}
+        self.attr['ALL'] = list(self.videos.keys())
+        for x in attr:
+            self.attr[x] = []
+        for k, v in self.videos.items():
+            for attr_ in v.attr:
+                self.attr[attr_].append(k)
+
+if __name__ == '__main__':
+    OTBDataset(name='OTB2015',dataset_root='/home/ubuntu/cyk/tracking/tracking_test_2/testing_dataset/OTB2015/',load_img=False)
```

Comment:
It is better to remove specific path on your machine, such as `/home/ubuntu/cyk/tracking/tracking_test_2/testing_dataset/OTB2015/`. Usually we ignore it or we use a default location at `~/.mxnet/datasets/`.